jenkins/common/sonic-utilities-build-pr/Jenkinsfile

```diff
@@ -36,7 +45,10 @@ pipeline {
         stage('Test') {
             steps {
                 wrap([$class: 'AnsiColorBuildWrapper', 'colorMapName': 'xterm']) {
-                    sh './scripts/common/sonic-utilities-build/test.sh'
+                    /* We will only run SwSS tests against master branch PRs */
+                    if ('${ghprbTargetBranch}' == 'master') {
```

Comment:
It this condition too narrow? 201911 could be skipped, I think we can still test future stable branches.

Reply:
I'll let @lguohan comment on this.